spec/lib/efile/nj/nj1040_spec.rb

```diff
@@ -21,33 +21,48 @@ def over_65_birth_year
       expect(instance.lines[:NJ1040_LINE_6].value).to eq(1000)
     end
 
-    context 'when filer is older than 65' do
-      let(:intake) do
-        create(:state_file_nj_intake,
-               primary_birth_date: Date.new(over_65_birth_year, 1, 1))
-      end
-
-      it 'checks the self 65+ checkbox and sets line 7 to 1000' do
-        expect(instance.lines[:NJ1040_LINE_7_SELF].value).to eq(true)
-        expect(instance.lines[:NJ1040_LINE_7_SPOUSE].value).to eq(false)
-        expect(instance.lines[:NJ1040_LINE_7].value).to eq(1000)
+    context "when filer is not blind" do
+      it "sets line 8 deductions to 0" do
+        expect(instance.lines[:NJ1040_LINE_8].value).to eq(0)
       end
     end
 
-    context 'when filer is younger than 65' do
+    context "when filer is blind" do
```

Comment:
[pebble] this is the same exact test as "when filer is blind and spouse is not blind" I believe? That one is more specific wording-wise so I think this one should be removed

Reply:
Got it. Refactoring tests based on feedback above, will change.